Target-MVD-Mathias/Networking/Services/TargetAPI.swift

```diff
@@ -81,4 +81,32 @@ class TargetAPI {
     }
   }
   
+  class func updateTarget(target: Target, success: @escaping (_ target: Target, _ compatibleTargets: [Target]) -> Void, failure: @escaping (_ error: Error) -> Void) {
+    let url = usersUrl + "\(UserDataManager.getUserId())" + targetsUrl + "\(target.id)"
+    
+    let params = [
+      "lat": target.lat,
+      "lng": target.lng,
+      "title": target.title,
+      "radius": target.radius,
+      "topic_id": target.topic.id
+    ] as [String : Any]
+    
+    APIClient.sendPutRequest(url, params: params as [String : AnyObject], success: { (response) in
+      let json = JSON(response)
+      let target = Target.parse(fromJSON: json["target"])
+      
+      let compatibleTargetsArray = json["matches"].arrayValue
+      var compatibleTargets: [Target] = []
+      
+      for compatibleTarget in compatibleTargetsArray {
```

Comment:
Parsing a lot of Targets at once seems like something you will have to do more than once, you should consider moving this to the Target model and make a general Target array parser.

Reply:
Done!